src/commands.py

```diff
@@ -540,3 +545,27 @@ async def say(message: discord.Message, _):
             await message.channel.send("You do not have permissions to post in that channel.")
         else:
             await message.channel.send(f"Oh god something went wrong, everyone panic! {str(err)}")
+
+
+"""
+_get_am
+
+Returns the answering machine to use for a command.
+
+Figures out which one to use given whether threads are on and the channel the message was sent in.
+
+If the option to turn off reply threads is removed, this isn't needed and we can just use the reply_am/delete the ban_am.
+"""
+async def _get_am(channel: discord.TextChannel | discord.Thread) -> AnsweringMachine | None:
+    # If reply threads are on, everything goes under mailbox using the regular answering machine
+    if FORWARDING_CREATE_THREADS:
+        return reply_am
+
+    # Otherwise it depends on the channel (or existing thread created before threads were turned off) where the command is used
+    if channel.id == BAN_APPEAL or (isinstance(channel, discord.Thread) and channel.parent.id) == BAN_APPEAL:
```

Comment:
I think the parenthesis look wrong here

Reply:
Wow, you're right - good catch.

And yet, the code still works as expected:
*  `True and 5` evaluates to `5` - so the thread id comparison still works.
* `False and 5` evaluates to `False` - so the thread comparison fails, as it should. *And* due to short circuiting it doesn't evaluate `5`, which would fail in the real code because `channel.parent` would not be defined.

Anyways, fixed!